client/me/help/help-contact-closed/index.jsx

```diff
@@ -6,35 +6,152 @@
+					'and we will reopen live chat on Monday, October 8th.{{/p}}',
+				{ components: { p: <p /> } }
+			)
+		);
+	}
+
+	if ( ! compact ) {
+		message.push(
+			translate(
+				'{{p}}Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family ' +
+					'get together to work on improving our services, building new features, and learning how to better serve you, ' +
+					'our users. But never fear! If you need help in the meantime, you can submit an email ticket through the contact form.{{/p}}',
+				{ components: { p: <p /> } }
+			)
 		);
 	}
 
+	return message;
+};
+
+const PremiumAndPersonalPlanMessage = ( { compact, translate } ) => {
+	const message = [];
+
+	if ( i18n.moment() < gm2018ClosureStartsAt ) {
+		message.push(
+			translate(
+				'{{p}}Live chat support will be closed from Saturday, September 29th through Sunday, October 7th, included. Email support will be open during that time, and we will reopen live chat on Monday, October 8th.{{/p}}',
+				{ components: { p: <p /> } }
+			)
+		);
+	} else {
+		message.push(
+			translate(
+				'{{p}}Live chat support will be closed through Sunday, October 7th, included. We will reopen private support on Sunday, October 7th.{{/p}}',
+				{ components: { p: <p /> } }
+			)
+		);
+	}
+
+	if ( ! compact ) {
+		message.push(
+			translate(
+				'{{p}}Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family ' +
+					'get together to work on improving our services, building new features, and learning how to better serve you, ' +
+					'our users. But never fear! If you need help in the meantime, you can submit an email ticket through the contact form.{{/p}}',
+				{ components: { p: <p /> } }
+			)
+		);
+	}
+
+	return message;
+};
+
+const GeneralMessage = ( { compact, translate } ) => {
+	const message = [];
+
+	if ( i18n.moment() < gm2018ClosureStartsAt ) {
+		message.push(
+			translate(
+				'{{p}}Private support will be closed Saturday, September 29th through Saturday, October 6th, included. We will reopen private support on Sunday, October 7th.{{/p}}',
+				{ components: { p: <p /> } }
+			)
+		);
+	} else {
+		message.push(
+			translate(
+				'{{p}}Private support will be closed through Saturday, October 6th, included. We will reopen private support on Sunday, October 7th.{{/p}}',
+				{ components: { p: <p /> } }
+			)
+		);
+	}
+
+	if ( ! compact ) {
+		message.push(
+			translate(
+				'{{p}}Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family ' +
+					'get together to work on improving our services, building new features, and learning how to better serve you, ' +
+					'our users. But never fear! If you need help in the meantime:{{/p}}' +
+					'{{p}}{{supportLink}}https://en.support.wordpress.com{{/supportLink}}{{/p}}' +
+					'{{p}}Our staff will be keeping an eye on the {{forumLink}}Forums{{/forumLink}} for urgent matters.{{/p}}',
```

Comment:
This is a bit unfortunate for translation because in many languages it's not true.